app/controllers/api/v1/reference/regions_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Reference
+      class RegionsController < ApiController
+        def index
+          render json: Region.all.includes(:locations)
```

Comment:
do we need to paginate this as well?

Reply:
I don't think it's necessary, there are only 9 regions in total - and that's supposedly going to reduce to two in future. If instead it increases beyond 20 we can look at adding pagination then.